### ISSUE TITLE
Commitwatch sometimes picks up merged PRs' commit messages and tries to format them

### ISSUE THREAD

**Issue by mcataford** [tophat/commit-watch#18]
In some cases, CW will inexplicable pick up on commit messages from commits that are already merged in `master`. Since the commit message format isn't enforced beyond branch pushes, this breaks the formatting function as one would expect.

![image](https://user-images.githubusercontent.com/6210361/76024080-7a00e200-5ef8-11ea-9c2e-ca97f617b780.png)

This problem is intermittent and couldn't be reliably reproduced, but it happens every once and again. This hints at a yet-unexplored gap in how CW builds the list of messages it'll lint.

**Comment by msrose** [comment]
I don't think this has to do with picking up commits from the main branch, based on the error message. It seems like certain commits will cause the format function to fail, but it can happen to any commit it seems. Commit watch also doesn't control which commits get linted as far as I understand, but I'm not entirely familiar with it.

**Comment by msrose** [comment]
I've discovered the immediate problem and will put up a fix soon

**Comment by msrose** [comment]
Addressed the immediate error in #19 but will leave this open because there will likely be other errors related to linting incorrect messages
